### PR TITLE
fr translation for project slogan

### PR DESCRIPTION
**Pull request by sftim**
Amend translation for “Production-Grade Container Orchestration”. Also, revise abstract for French top level page on site. [[preview](https://deploy-preview-18267--kubernetes-io-master-staging.netlify.com/fr/)]

/language fr
/kind cleanup

Hope this is OK to target _master_ branch.

### CODE REVIEW

File: content/fr/_index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "La meilleure solution d'orchestration de conteneurs en production"
+title: "Solution d'orchestration de conteneurs, prête pour production"
 abstract: "Déploiement, mise à l'échelle et gestion automatisés des conteneurs"
```
Comment (mhamdisemah) [inline content/fr/_index.html]: abstract: "Déploiement, mise à l'échelle et gestion automatisée des conteneurs"

Comment (sftim) [reply content/fr/_index.html]: @mhamdisemah I can do a new pull request for that. This PR is just about the slogan.

Comment (sftim) [reply content/fr/_index.html]: I've revised

**Comment by remyleone** [comment]
@sftim Could you develop about why the current slogan needs change?

**Comment by sftim** [comment]
> @sftim Could you develop about why the current slogan needs change?

This change @remyleone: https://github.com/kubernetes/website/pull/18267#discussion_r361114722 ?

Or the change from “Production-Grade Container Orchestration” to “Solution d'orchestration de conteneurs, prête pour production”?

**Comment by remyleone** [comment]
I would say both. I don't understand what is wrong with the current slogan. Also, "Solution d'orchestration de conteneurs, prête pour production” sounds weird. 

A proposal from @perriea would be: "Solution d'orchestration de conteneurs, prête pour la production”
A proposal from @Sh4d1 would be: "Solution d'orchestration de conteneurs, prête à être utilisée en production"

**Comment by sftim** [comment]
> A proposal from @perriea would be: "Solution d'orchestration de conteneurs, prête pour la production”
A proposal from @Sh4d1 would be: "Solution d'orchestration de conteneurs, prête à être utilisée en production"

Which should I pick? I don't mind.

**Comment by sftim** [comment]
BTW, it seems obvious to me but for clarity the reason for replacing “Production-Grade Container Orchestration” is that it's not localized. The reason for other changes around this slogan is to use one (localized) slogan consistently.

**Comment by AWKIF** [comment]
i vote for @Sh4d1 :)
or my 2 cents:
"Solution d'orchestration de conteneurs utilisable en environnement de production"

**Comment by remyleone** [comment]
Solution professionnelle d’orchestration de conteneurs. Seems to be a great option. I think we should go with this one. It sounds more idiomatic IMHO. @AWKIF @perriea @Sh4d1

**Comment by Sh4d1** [comment]
works fo me :heavy_check_mark:

**Comment by remyleone** [comment]
@sftim Could you update to "Solution professionnelle d’orchestration de conteneurs"?

**Comment by sftim** [comment]
> @sftim Could you update to "Solution professionnelle d’orchestration de conteneurs"?
Sure, should have time later.

**Comment by sftim** [comment]
@remyleone @AWKIF @perriea @Sh4d1 please take a look and /lgtm if happy

**Pull request merged**